### ISSUE TITLE
Add package creation to tests

### ISSUE THREAD

**Issue by nyggus** [nyggus/makepackage#23]

**Comment by patrickboateng** [comment]
So I guess the package creation will be done in a separate test right?
And I am guessing the whole design will be:

1. Create a virtual environment.
1. Install the package in a virtual environment.

**Comment by nyggus** [comment]
Yes. But I am not sure these should be separate tests, as in order to run such a test, you need to create the package. So, this should be tested in one run, so in the current test functions.

These will become impressive integration tests, but as I wrote elsewhere, we will also implement unit tests.
